lib/cog/util/ecto.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Cog.Util.Ecto do
+
+  require Logger
+
+  @doc """
+  Sanely logs Ecto query output
+  """
+  def log(entry) do
+    IO.puts("here")
```

Comment:
Argh. Yeah. That's from a failed experiment to tame Ecto's verbose logging. Sadly, it only works with 2.0 :(